includes/polling/ports.inc.php

```diff
@@ -227,7 +227,7 @@
 if ($config['enable_ports_etherlike']) {
     echo 'dot3Stats ';
     $port_stats = snmpwalk_cache_oid($device, 'dot3StatsEntry', $port_stats, 'EtherLike-MIB');
-} else {
+} elseif ($device['os'] != 'asa') {
```

Comment:
Maybe, don't want to end up with tons of modules/submodules.
It does seem to make sense in this circumstance.